doc/src/DebugAndLogging.md

```diff
@@ -135,6 +135,7 @@ Winograd  Solutions:
   * `MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F4X3` - `ConvMPBidirectWinograd_xdlops<4-3>`, FWD/BWD F(4,3)
   * `MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F5X3` - `ConvMPBidirectWinograd_xdlops<5-3>`, FWD/BWD F(5,3)
   * `MIOPEN_DEBUG_AMD_MP_BD_XDLOPS_WINOGRAD_F6X3` - `ConvMPBidirectWinograd_xdlops<6-3>`, FWD/BWD F(6,3)
+  * `MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_ENABLE_FP16_TRANSFORM` - `ConvMPBidirectWinograd*`, FWD/BWD FP16 experemental mode.
```

Comment:
Typo in "experemental". Use the "pure fp16" expression.
```suggestion
  * `MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_PURE_FP16 - `ConvMPBidirectWinograd*`, FWD/BWD pure FP16 mode. Disabled by default. This mode is experimental. Use it at your own risk.
```

Reply:
I will add everything, except 'pure FP16 mode' because it is not 'pure'.